.github/workflows/ci.yml

```diff
@@ -46,3 +46,34 @@ jobs:
           install-command: yarn --frozen-lockfile
       - name: Yarn build
         run: yarn build
+  lint-test-auth:
+    name: Lint and test auth package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # This clones the repo and checks out the SHA that triggered this action.
+          # We set fetch-depth 0 to fetch all branches and history so that merge-base
+          # is guaranteed to be able to find the common ancestor with the base branch.
+          # See https://github.com/actions/checkout#readme
+          fetch-depth: 0
+      - name: Yarn caching
+        uses: actions/cache@v2
+        id: yarn-caching
+        with:
+          path: "**/node_modules"
+          key: yarn-v2-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            yarn-v2-${{ hashFiles('**/yarn.lock') }}
+            yarn-v2-
+            yarn-
+      - name: Yarn install
+        if: steps.yarn-caching.outputs.cache-hit != 'true'
+        working-directory: packages/auth
+        run: yarn install --frozen-lockfile
+      - name: Yarn lint
+        working-directory: packages/auth
+        run: yarn lint
+      - name: Yarn test
+        working-directory: packages/auth
```

Comment:
I think it's better actually to separate `lint` and `test` into two different jobs so that they can fail independently of one another. I know we have one mega-job for all frontend stuff in `recidiviz-data` but I think that was emulating some older patterns in the CI config that we have been moving away from

Reply:
not that it's rocket science or anything but you can see examples of this in `public-dashboard`

Reply:
sorry for spamming this thread but I thought of something else! also a good idea to add a CI build task on every push (as we do with the design system above) so that you don't merge a branch that won't build and not find out until someone tries to publish it

Reply:
That makes sooo much sense. Thanks for this redirect!

Reply:
OK - added a build step and separated the lint and and test steps. Thank you for pointing this out.